src/documents/classifier.py

```diff
@@ -442,10 +512,8 @@ def predict_document_type(self, content: str) -> int | None:
             return None
 
     def predict_tags(self, content: str) -> list[int]:
-        from sklearn.utils.multiclass import type_of_target
```

Comment:
Imports like these were deliberate.  Not every path needed them, so it's a minor optimization to import only when needed